app/src/main/assets/init.js

```diff
@@ -173,6 +173,21 @@ try {
 
 window.addEventListener('file-received', e => {
     SnapdropAndroid.saveDownloadFileName(e.detail.name, e.detail.size);
```

Comment:
I wouldn't recommend this as the device vibrates multiple times for a single transfer which is annoying.